src/Chartjs/Resources/assets/src/controller.ts

```diff
@@ -38,9 +38,10 @@ export default class extends Controller {
         this._dispatchEvent('chartjs:connect', { chart });
     }
 
-    _dispatchEvent(name: string, payload: any = null, canBubble = false, cancelable = false) {
-        const userEvent = document.createEvent('CustomEvent');
-        userEvent.initCustomEvent(name, canBubble, cancelable, payload);
+    _dispatchEvent(name: string, payload: any) {
+        const userEvent = new CustomEvent(name, {
```

Comment:
I guess we don't need this structure anymore do we? Perhaps could we replace all `this._dispatchEvent(...)` calls by `this.element.dispatchEvent(new CustomEvent(...));` directly?